core/src/main/java/io/questdb/cairo/frm/file/ContiguousFileIndexedFrameColumn.java

```diff
@@ -59,20 +57,22 @@ public void append(long offset, FrameColumn sourceColumn, long sourceLo, long so
                     indexWriter.add(TableUtils.toIndexKey(Unsafe.getUnsafe().getInt(mappedAddress + (i << shl))), offset + i);
                 }
                 indexWriter.setMaxValue(offset + size - 1);
+                indexWriter.commit();
             } finally {
                 TableUtils.mapAppendColumnBufferRelease(ff, mappedAddress, (offset - getColumnTop()) << shl, size << shl, MEMORY_TAG);
             }
         }
     }
 
     @Override
-    public void appendNulls(long offset, long count) {
-        super.appendNulls(offset, count);
+    public void appendNulls(long offset, long count, int commitMode) {
+        super.appendNulls(offset, count, commitMode);
         indexWriter.rollbackConditionally(offset);
         for (long i = 0; i < count; i++) {
             indexWriter.add(0, offset + i);
         }
         indexWriter.setMaxValue(offset + count - 1);
+        indexWriter.commit();
```

Comment:
nit: commit mode is ignored

Reply:
same comment as this: https://github.com/questdb/questdb/pull/3304#discussion_r1192575225